src/dropbox.js

```diff
@@ -206,5 +215,10 @@ export default class Dropbox {
     if (this.pathRoot) {
       options.headers['Dropbox-API-Path-Root'] = this.pathRoot;
     }
+    if (this.customHeaders) {
+      for (const header in this.customHeaders) {
```

Comment:
linter is failing from this line but I think this is actually the behavior we more or less want, can alter the code to satisfy linter but I feel like it just adds extra code for no reason. Can also ignore lint this line if we want

Reply:
Valid linter, we don't want to do this